srcpkgs/httpie/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'httpie'
 pkgname=httpie
-version=2.2.0
-revision=2
+version=2.5.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
```

Comment:
```suggestion
hostmakedepends="python3-setuptools python3-pip python3-wheel"
```

Reply:
Why are you bringing pip into this? System packages shouldn't use it

Reply:
I was trying to fix failing tests saying `pip not found`.
It seems there was progress, because now failing jobs are talking about missing `pytest`. But it should not be related to my changes, right?

Are failures expected for some configuration?

Reply:
pip is used to install packages that are missing. This means the environment in which the tests will run will **not** be similar to when a user installs this package, so it's useless to run tests like this. We should either specify the check dependencies in `$checkdepends` or disable tests.

In this case, if we add
```sh
checkdepends="python3-pytest $depends"
```
it will fail with
```
ImportError while loading conftest '/builddir/httpie-2.5.0/tests/conftest.py'.
tests/conftest.py:3: in <module>
    from pytest_httpbin import certs
E   ModuleNotFoundError: No module named 'pytest_httpbin'
```

pytest_httpbin is not packaged, let's disable tests and add a comment about this, so if we package it in the future, we can enable tests:

```sh
make_check=no # needs pytest_httpbin which is not packaged
```

Reply:
Thanks for the explanation @paper42.